File: java_programming/02_Variable/ex/Ex01.java
package ex01;

public class Ex01 {

	public static void main(String[] args) {
		
		int noodleCup = 0;

		noodleCup = 850;

		System.out.println("육개장 가격 : " + noodleCup);

		// 세개 가격 계산
		int sum = 0;
		sum = noodleCup * 3;
		System.out.println("육개장 3개의 가격 : " + sum);

		// 가격 상승
		noodleCup = 950;

		sum = noodleCup * 3;
		System.out.println("육개장 가격 : " + noodleCup + ", 3개의 가격 : " + sum);
		
		System.out.print("육개장 가격 : " + noodleCup);
		System.out.println(", 3개의 가격 : " + sum);

	}

}

File: c_programming/02_Variable/ex/ex01.c
#include <stdio.h>

int main(int argc, const char * argv[]) {
    
    int noodleCup = 0;
    
    noodleCup = 850;
    
    printf("육개장 가격은 %d 입니다.\n", noodleCup);
    
    //  세개 가격 계산
    int sum = 0;
    sum = noodleCup * 3;
    printf("육개장 3개의 가격은 %d 입니다.\n", sum);
    
    // 가격 상승
    noodleCup = 950;
    
    sum = noodleCup * 3;
    printf("육개장 가격은 %d 이고, 3개의 가격은 %d 입니다.\n", nudleCup, sum);
    
    return 0;
    
}
